sycl/doc/extensions/PropertyList/SYCL_EXT_ONEAPI_property_list.asciidoc

```diff
@@ -226,12 +242,13 @@ and type of the parameter. When a property has more than one parameter, the
 the values and types of the parameters.
 
 ```c++
-namespace sycl {
-namespace ext {
-namespace oneapi {
+namespace sycl::ext::oneapi {
 
-template<class property, class first, class...others>
+template<typename property, typename first, typename...others>
```

Comment:
Nit: I think capitalizing "Property", "First" and "Others" would be more consistent with SYCL conventions, but I don't want to block this PR for this.